lib/foreman_bootdisk/engine.rb

```diff
@@ -154,4 +155,11 @@ class Engine < ::Rails::Engine
   def self.logger
     Foreman::Logging.logger('foreman_bootdisk')
   end
+
+  def self.with_proxmox?
+    ForemanFogProxmox # rubocop:disable Lint/Void
+    true
+  rescue StandardError
```

Comment:
I would expect this to be a `NameError`.

Alternatively: `Object.const_defined?('ForemanFogProxmox')` might work as well